src/client.js

```diff
@@ -340,7 +340,7 @@ Client.prototype.names = function(data) {
 	}
 
 	client.emit("names", {
-		chan: target.chan.id,
```

Comment:
Oops, I see what you mean now. Yeah we should just fix it in a separate PR to use `chan` everywhere then.